config/rector.php

```diff
@@ -11,9 +16,39 @@
             __DIR__ . '/../tests',
         ]
     )
-    ->withPhpSets()
-    ->withRules(
-        [
-            // AddVoidReturnTypeWhereNoReturnRector::class,
-        ]
-    );
+    ->withPhpVersion(PhpVersion::PHP_73)
```

Comment:
[The best practise is to let [it] pickup the PHP version from composer.json](https://getrector.com/documentation/php-version-features).  So I don't think we need this (as it will be something more to update), unless it doesn't work as expected and desired otherwise.

Reply:
Ah, I wasn't aware if of this behavior. Removed now.